Tests/conf.json

```diff
@@ -5787,7 +5784,6 @@
         "FeedMitreAttackv2",
         "URLHaus",
         "MicrosoftDefenderAdvancedThreatProtection",
```

Comment:
Cant we add the pack you added?

Reply:
do we need to? not sure what this is for.